### ISSUE TITLE
Missing course after upgrade of servers:  MCSP_21-22

### ISSUE THREAD

**Issue by suesoucy** [RunestoneInteractive/RunestoneServer#1839]
**Please check the current issues** Many bug reports are duplicates, this just creates more work for us. Searching the issues first may give you your answer or a workaround! If not adding new information to an existing report is much more helpful than a new report.

**What Course are you in**
Knowing the course will help us a lot

**What Page were you on**

**What is your username**

**Describe the bug**
A clear and concise description of what what you were trying to accomplish and what went wrong. Vague statements like X does not work are not helpful. Imagine if one of your students sent you an email that said, "My program doesn't work, whats wrong with it?" Could you help them with only that to go on?

**Traceback**
If you got here through the Bug Report page and there was a stack trace, please paste it here

**Javascript Errors**
Many many errors can be quickly diagnosed from the Javascript Console error log. On Chrome this is View --> Developer --> Javascript Console, on Safari ⌥⌘i or Develop --> Show Web Inspector on Firefox Tools --> Web Developer --> Toggle Tools

**Comment by bnmnetp** [comment]
mobilcsp courses are all there. can you please provide a little more detail on the problem.

**Comment by suesoucy** [comment]
When I click on the course it just brings me to the runestone homepage, not
the course.

**Comment by bnmnetp** [comment]
are you logged in?

**Comment by suesoucy** [comment]
Yes I was logged in. I could get to all my other courses.  I could also get
to the instructors page of the MCSP_20-21, and when I clicked on the
student progress tab and clicked on the chapter activity button it gave me
an error. I just logged out and went back in again and I was able to get to
it from the course page and I was also able to see the chapter activity.
Not sure why all the other courses worked except this one, but it seems to
be all set after logging out and back in again. Thank you for your help.
